### PR TITLE
Change user model to integer

### PR DESCRIPTION
**Pull request by CSekwalor**

### CODE REVIEW

File: rest_framework_tracking/base_mixins.py [hunk 1]
```diff
@@ -157,10 +155,15 @@ def _get_view_method(self, request):
 
     def _get_user(self, request):
         """Get user."""
-        user = request.user
```
Comment (CSekwalor) [inline rest_framework_tracking/base_mixins.py]: Why remove this and duplicate code

File: rest_framework_tracking/admin.py [hunk 2]
```diff
@@ -28,9 +28,8 @@ class APIRequestLogAdmin(admin.ModelAdmin):
     list_filter = ("view_method", "status_code")
     search_fields = (
         "path",
-        "user__email",
+        "user",
     )
-    raw_id_fields = ("user",)
```
Comment (CSekwalor) [inline rest_framework_tracking/admin.py:33]: What does this do

File: rest_framework_tracking/base_mixins.py (hunk 1 above)
Comment (maiasoliman) [reply rest_framework_tracking/base_mixins.py]: this returns user model

File: rest_framework_tracking/admin.py (hunk 2 above)
Comment (maiasoliman) [reply rest_framework_tracking/admin.py:33]: this caused migration error


ERRORS:
<class 'rest_framework_tracking.admin.APIRequestLogAdmin'>: (admin.E003) The value of 'raw_id_fields[0]' must be a foreign key or a many-to-many field.
<class 'rest_framework_tracking.admin.APIRequestLogAdmin'>: (admin.E003) The value of 'raw_id_fields[0]' must be a foreign key or a many-to-many field.

**Pull request merged**